### PR TITLE
[ko] Update outdated files in `dev-1.24-ko.1` (M1-M16)

### PR DESCRIPTION
**Pull request by jihoon-seo**
한 PR에 너무 많은 변경사항을 포함시키면 리뷰하기 어려울 수 있어서, 적절히 분할하여 올립니다. 😊

- Related issue: #33472 (**dev-1.24-ko.1**)

> ### 16 files to be modified
> 1. [ ]  M1.  `content/en/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md`  | 12(+S) 4(-)
> 2. [ ]  M2.  `content/en/blog/_posts/2021-08-04-kubernetes-release-1.22.md`  | 1(+XS) 0(-)
> 3. [ ]  **M3.**  `content/en/community/_index.html`  | **183(+L) 257(-)**
> 4. [ ]  M4.  `content/en/community/code-of-conduct.md`  | 9(+XS) 7(-)
> 5. [ ]  M5.  `content/en/community/static/README.md`  | 4(+XS) 1(-)
> 6. [ ]  M6.  `content/en/docs/concepts/architecture/garbage-collection.md`  | 1(+XS) 1(-)
> 7. [ ]  **M7.**  `content/en/docs/concepts/architecture/nodes.md`  | **123(+L) 42(-)**
> 8. [ ]  M8.  `content/en/docs/concepts/cluster-administration/_index.md`  | 1(+XS) 1(-)
> 9. [ ]  M9.  `content/en/docs/concepts/cluster-administration/addons.md`  | 2(+XS) 2(-)
> 10. [ ]  M10.  `content/en/docs/concepts/cluster-administration/logging.md`  | 12(+S) 2(-)
> 11. [ ]  M11.  `content/en/docs/concepts/cluster-administration/manage-deployment.md`  | 1(+XS) 1(-)
> 12. [ ]  M12.  `content/en/docs/concepts/cluster-administration/networking.md`  | 1(+XS) 1(-)
> 13. [ ]  M13.  `content/en/docs/concepts/cluster-administration/system-logs.md`  | 50(+M) 21(-)
> 14. [ ]  M14.  `content/en/docs/concepts/configuration/configmap.md`  | 2(+XS) 4(-)
> 15. [ ]  M15.  `content/en/docs/concepts/configuration/manage-resources-containers.md`  | 7(+XS) 8(-)
> 16. [ ]  M16.  `content/en/docs/concepts/configuration/organize-cluster-access-kubeconfig.md`  | 21(+S) 0(-)

/language ko

### CODE REVIEW

File: content/ko/docs/concepts/cluster-administration/system-logs.md [hunk 1]
```diff
@@ -110,6 +110,55 @@ I1025 00:15:15.525108       1 example.go:116] "Example" data="This is text with
 second line.}
 ```
 
+### 컨텍스츄얼 로깅
```
Comment (seokho-son) [inline content/ko/docs/concepts/cluster-administration/system-logs.md:113]:
```suggestion
### 컨텍스츄얼 로깅(Contextual Logging)
```

영문 병기하면 좋을 것 같습니다.

**Review by seokho-son — COMMENTED** [review]
@jihoon-seo

**Comment by jihoon-seo** [comment]
@seokho-son 리뷰 감사합니다. 해당 영문 병기 반영하고 squash까지 하였습니다. 😊

**Pull request merged**